### ISSUE TITLE
Emojis not showing up in coverage report

### ISSUE THREAD

**Issue by gigiskarlett** [ArtiomTr/jest-coverage-report-action#116]
Hi, using the GH action to print the coverage report I can't see the emojis/icons for `-ok` and `-slightly more than threshold`.
Not sure if this is only a problem I see but this is how it looks on Chrome and Firefox.

I'm using macOS Mojave Version 10.14.6.

Chrome
![Screen Shot 2021-07-16 at 1 04 48 PM](https://user-images.githubusercontent.com/38567356/126003493-3f40978d-5ba9-4e8e-936a-1e3c1b1076fc.png)

Firefox
<img width="956" alt="Screen Shot 2021-07-16 at 1 11 31 PM" src="https://user-images.githubusercontent.com/38567356/126003522-7537a577-8a9a-47b2-836f-e9b64aefee24.png">

Thanks!

**Comment by ArtiomTr** [comment]
Hello @gigiskarlett! 👋

Sorry for not being able to answer you promptly! If you have dealt with the problem yourself, then just ignore my answer. I'll leave it here for other people who might faced same issue and want to know, how to fix this.

Unfortunately, there is no competent solution for this problem. This problem was already mentioned earlier (#41), and I found the only solution that will help in this situation - to use other icons (solved by this PR #42). 

## Short answer

If you face the same issue, try to change icons by using this configuration:
```yml
name: 'coverage'
on:
    pull_request:
        branches:
            - master
jobs:
    coverage:
        runs-on: ubuntu-latest
        env:
            CI_JOB_NUMBER: 1
        steps:
            - uses: actions/checkout@v1
            - uses: artiomtr/jest-coverage-report-action@v2.0-rc.2
              with:
                  github_token: ${{ secrets.GITHUB_TOKEN }}
#                         ↓
                  icons: emoji
```

Hope my answer will help you!

**Comment by gigiskarlett** [comment]
Thank you for your answer, it seems to be related to the OS version since my coworkers are able to see the emojis.
